index.html

```diff
@@ -1005,10 +1011,10 @@ <h3>Data delivery</h3>
       are as follows:
       <ol>
         <li>
-          Let |source:PressureSource| be the [=source type=] of the |data| sample.
+          Let |source:PressureSource| be the [=source type=] of the [=implementation-defined=] |data| sample.
```

Comment:
I think no need to repeat "implementation-defined" on L1003, better keep that keyword on this line in these steps.

Reply:
done